hack/update_hash.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2020 Red Hat, Inc.
+#
+
+# Usage example: ./hack/update_hash.sh k8s-1.18 HASH
+
+LANE=${1:?}
```

Comment:
it doesn't look like you're using these env params, and instead using the $1,$2 in the main function... did you mean to use them in the main function?

Reply:
ye you right, i want to use them, it works because those two lines enforce them to exists, but i would change it
thanks

Reply:
done